lib/auth/methods.go

```diff
@@ -216,6 +216,7 @@ type AuthenticateSSHRequest struct {
 	TTL time.Duration `json:"ttl"`
 	// CompatibilityMode sets certificate compatibility mode with old SSH clients
 	CompatibilityMode string `json:"compatibility_mode"`
+	RouteToCluster    string `json:"route_to_cluster"`
```

Comment:
Question regarding migration, is it okay for this value to be empty? If you have an old client connecting to a new server, this value will not be set.

Reply:
Yes, if this field is empty it will not be populated in the TLS cert (in the `street` field).
Before this change, it was never populated.
Confirmed locally with newer server and older client.